configure

```diff
@@ -46,6 +46,7 @@ CLOUSEAU_URI=${CLOUSEAU_URI:-"https://github.com/cloudant-labs/clouseau/releases
 CLOUSEAU_VSN=${CLOUSEAU_VSN:-"2.25.0"}
 CLOUSEAU_DIR="$(pwd)"/clouseau
 ARCH="$(uname -m)"
+MULTIARCH_NAME="$(dpkg-architecture -q DEB_HOST_MULTIARCH || true)"
```

Comment:
This is run unconditionally (on every OS), probably that is what causes `configure` to fail on macOS (and on other non-Debian systems).

Reply:
Yes, but it wouldn't break the build here. The fail on macOS was through a missing path. It will be fixed in #5657.